packages/twenty-front/src/modules/settings/data-model/fields/forms/select/utils/getOptionValueFromLabel.ts

```diff
@@ -7,7 +7,7 @@ export const getOptionValueFromLabel = (label: string) => {
   try {
     transliteratedLabel = computeOptionValueFromLabelOrThrow(label);
   } catch (error) {
-    return label;
+    return "";
```

Comment:
What are you trying to do here? You said you'd tackle "Stopped inputting special characters in Select field option keys" but that's not really what you're doing here?